notebooks/14-combine-data.md

```diff
@@ -138,11 +138,13 @@ def add_date_dimension(ds):
 
 ```{code-cell} ipython3
 moisture_index_lazy = xr.open_mfdataset(sorted(Path("./data/herstappe/raster/sentinel_moisture").rglob("*.tiff")), 
-                                        preprocess=add_date_dimension, engine="rasterio", decode_cf=False) # parallel=True
+                                        preprocess=add_date_dimension, engine="rasterio", decode_cf=True, 
+                                        chunks={"date": 1, "band": -1, "x": -1, "y": -1}) # parallel=True
+moisture_index_lazy = moisture_index_lazy.drop_vars("spatial_ref")
```

Comment:
Specific need for this? (to clean up the data to avoid having that extra coordinate without dimension? But it also adds something extra to the code to explain why it is there)

Reply:
readon is that otherwise the save to hdf5 file fails

Reply:
But apparently your solution works well as ell, all fine